Vizzuality/heco-invest
# Add Google Tag Manager and Hotjar tracking to the platform

This PR adds the Google Tag Manager and Hotjar tracking scripts to the platform. They are only initiated if the respective environment variable has a value.

The PR also documents both variables in the `README.md` file.

## Testing instructions

1. In your `.env`, add the following:
```
NEXT_PUBLIC_GOOGLE_ANALYTICS=hello-world
NEXT_PUBLIC_HOTJAR_SITE_ID=12345
```
2. Reload your local server

Make sure that Google Tag Manager and Hotjar are loaded. You could look at the browser's requests panel or at the scripts at the end of the DOM.

## Tracking

- [LET-1104](https://vizzuality.atlassian.net/browse/LET-1104): FE - Add GA tracking code
- [LET-1105](https://vizzuality.atlassian.net/browse/LET-1105): FE - Update the environment variables information with GA
- [LET-1101](https://vizzuality.atlassian.net/browse/LET-1101): FE - Add Hotjar tracking code
- [LET-1102](https://vizzuality.atlassian.net/browse/LET-1102): FE - Update the environment variables information with Hotjar
